src/main/java/io/vlingo/http/resource/ErrorHandler.java

```diff
@@ -0,0 +1,12 @@
+package io.vlingo.http.resource;
+
+import io.vlingo.common.Completes;
+import io.vlingo.http.Response;
+
+public interface ErrorHandler {
+  Completes<Response> handle(Throwable error);
```

Comment:
The type here should be Exception; otherwise it leads the developer to believe that Errors could be handled by the handle() implementation.